lib/oelite/util.py

```diff
@@ -57,6 +57,11 @@ def write(self, text):
             file.write(text)
         return
 
+def stracehack(msg):
```

Comment:
shouldn't this be enabled by some global flag? It might confuse unaware users should they happen to debug oe-lite with strace too...